pages/en/download/package-manager/all.md

```diff
@@ -1,5 +1,5 @@
 ---
-layout: docs
+layout: download
```

Comment:
Afaik the layout intentionally was docs so that there was a limit of width for the content, but completely fine with this hotfix!